PG/Orders/index.js

```diff
@@ -10,135 +9,128 @@ const {
     ORDER_EMAIL_REL_URL,
 } = require('../constants');
 
-const CreateOrders = async (data = {}) => {
-    try {
-        const pgConfig = getPGConfig();
-        // make request
-        const res = await doPost({
-            hostname: pgConfig.baseUrl,
-            path: `${PG_REL_URL}${PG_API_VERSION}${ORDER_CREATE_REL_URL}`,
-            headers: {
-                'Content-Type': 'application/x-www-form-urlencoded',
-            },
-            data: {
-                appId: pgConfig.appId,
-                secretKey: pgConfig.secretKey,
-                ...data,
-            },
-        });
-        return res;
-    } catch (e) {
-        return {
-            status: 'ERROR',
-            reason: e && e.message || 'Something went wrong',
-        };
-    }
-};
+function Orders() {
+    const { baseUrl: hostname, appId, secretKey } = this;
+    return {
+        CreateOrders: async (data = {}) => {
```

Comment:
Except modules and constructors, all methods should be camelCase.

Reply:
I followed the nomenclature from Payouts. I will have to ask to Raj for this change.